uk.co.compucorp.civicrm.hrcore/CRM/HRCore/Form/CreateUserRecordTaskForm.php

```diff
@@ -25,8 +26,7 @@ public function buildQuickForm() {
   public function preProcess() {
     parent::preProcess();
 
-    $haveNoAccount = $this->getContactsWithoutAttribute('uf_id');
-    $haveAccount = array_diff_key($this->contactDetails, $haveNoAccount);
+    $haveAccount = $this->getContactsWithAccount();
```

Comment:
Is there a reason to create this variable? If not, for consistency, can you just assign the value for the form by calling the getContactsWithAccount() method?

```php
$this->assign('contactsWithAccount', $this->getContactsWithAccount());
```

Reply:
You're right, it was used before but it doesn't need to be separate here.